gym/lib/gym/code_signing_mapping.rb

```diff
@@ -112,10 +119,13 @@ def detect_configuration_for_archive
       return configuration
     end
 
+    # rubocop:disable Metrics/PerceivedComplexity
     def detect_project_profile_mapping
       provisioning_profile_mapping = {}
       specified_configuration = detect_configuration_for_archive
 
+      is_building_mac_catalyst = Gym.project.mac_catalyst? && Gym.config[:catalyst_platform] == "macos"
```

Comment:
Add method onto `Gym` module like `Gym. is_building_mac_catalyst_for_mac?` and `Gym. is_building_mac_catalyst_for_ios?` but maybe better